jira/resources.py

```diff
@@ -482,6 +482,13 @@ def permalink(self):
     def __eq__(self, other):
         return self.id == other.id
 
+    def __getitem__(self, item):
+        # Grab the JIRA server
+        jira_server = self._session.jira_server
+
+        # Look for the appropriate fields list on the server
+        return getattr(self.fields, jira_server._fields[item])
```

Comment:
How about this: http://stackoverflow.com/questions/5843518/remove-all-special-characters-punctuation-and-spaces-from-string

Basically remove everything that isn't alphanumeric. I could add some code to be intelligent about avoiding duplicate names (i.e. appending "_X" on the end if necessary).